### DISCUSSION TITLE
On selecting the first suggested value the width size is incorrect

### DISCUSSION THREAD

**Discussion by AlfeeAeman** [JedWatson/react-select#5199] [Q&A]
<img width="1686" alt="image" src="https://user-images.githubusercontent.com/94458168/170010959-22bcc7f0-14cd-4db2-bce7-75aaa2a320e4.png">
When searching  and  selecting the first suggested value from the options, the style is incorrect.
<Select
      components={animatedComponents}
      isMulti
      {...rest}
    />
On using animated components as props

**Comment by manjushsh** [comment]
@AlfeeAeman Any specific version?
It works just fine in this [Code Sandbox](https://codesandbox.io/s/react-select-multiple-forked-n0qpyg?file=/src/index.js)

**Reply by AlfeeAeman** [reply]
I am using 5.3.2

**Reply by AlfeeAeman** [reply]
this is specifically because i am using animated component.

**Reply by AlfeeAeman** [reply]
`import makeAnimated from "react-select/animated";
const animatedComponents = makeAnimated();
<Select
          components={animatedComponents}
          isMulti
          {...rest}
        />`

**Reply by manjushsh** [reply]
looks like a bug

**Comment by manjushsh** [comment]
@AlfeeAeman, you can use react-select v.5.1.0 if that works for you. 
It seems have this issue from 5.2.0 onward. Use 5.0.0 if you see any animation issues.
It works fine in this version. [Code Sandbox](https://codesandbox.io/s/react-select-multiple-forked-n0qpyg?file=/src/index.js)

**Reply by AlfeeAeman** [reply]
Yes this version solves the problem of styling but it has other issue for example the placement to input cursor is after the placeholder value
<img width="662" alt="image" src="https://user-images.githubusercontent.com/94458168/170444606-f29b02d8-0156-41ca-b7f3-5ee96d8995a1.png">

**Reply by manjushsh** [reply]
@AlfeeAeman, In 5.1 it seems to be working.. but not ideal solution though..
![sample](https://user-images.githubusercontent.com/94426452/170451592-6c7b9f9e-119c-4fa3-9f2d-857dfa7decf4.png)

And also, there is issue of textbox size in clear animation in this version. I have created a bug for the same.

**Comment by imliuyzh** [comment]
Can confirm this. So far, I must either get rid of animations in the component or use `!important` in the styling:
```
const styles = {
    valueContainer: (styles) => ({
        ...styles,
        '& div:first-of-type': {
            width: 'fit-content !important'
        }
    })
};
```
Since I want my code to be as clean as possible, guess I will just stop using animations.

**Reply by manjushsh** [reply]
@imliuyzh Raised this issue. Hope they will fix it in future releases https://github.com/JedWatson/react-select/issues/5205
